packages/web/src/components/data-entry/InputV2.tsx

```diff
@@ -92,30 +95,33 @@ export const InputV2 = (props: InputV2Props) => {
   )
 
   return (
-    <div className={cn(styles.root, style, className)}>
-      {elevatePlaceholder ? (
-        <label className={styles.elevatedLabel}>
-          <span
-            className={cn(styles.label, {
-              [styles.hasValue]: characterCount > 0
-            })}
-          >
-            {label}
-          </span>
-          {input}
-        </label>
-      ) : (
-        input
-      )}
+    <>
+      <div className={cn(styles.root, style, className)}>
+        {elevatePlaceholder ? (
+          <label className={styles.elevatedLabel}>
+            <span
+              className={cn(styles.label, {
+                [styles.hasValue]: characterCount > 0
+              })}
+            >
+              {label}
+            </span>
+            {input}
+          </label>
+        ) : (
+          input
+        )}
 
-      {showMaxLength && (
-        <div className={styles.characterCount}>
-          <span>
-            {characterCount}/{maxLength}
-          </span>
-        </div>
-      )}
-      {children}
-    </div>
+        {showMaxLength && (
+          <div className={styles.characterCount}>
+            <span>
+              {characterCount}/{maxLength}
+            </span>
+          </div>
+        )}
+        {children}
+      </div>
+      {helperText ? <HelperText error={error}>{helperText}</HelperText> : null}
```

Comment:
what's the intended use of helperText? Is it for validation?

Reply:
yeah exactly, using helper text here cause there are cases where we may want a description below an input, and it can double as an error message. if we just want to call it `errorText` in the future though im down

Reply:
wondering if we should have error and helper text separately so we avoid the conditionals on the caller end? though I guess error: true + helper text is also good

Reply:
yeah i can def see both ways, at least with material-ui they have helperText and error as separate things, so error state affects all things like the border, label, and helperText styles